src/Resources/views/FragmentAdmin/form.html.twig

```diff
@@ -1,14 +1,15 @@
 {% block form %}
-    {% set fragmentName = admin.fragmentServices[form.vars.data.type].name %}
+    {% set fragmentName = admin.fragmentServices[form.vars.data.type].name|trans({}, 'SonataArticleBundle') %}
+    {% set fragmentLabel = {{ 'fragment'|trans({}, 'SonataArticleBundle') }} %}
     <div data-fragment-form="{{ form.vars.id }}"
          data-form-tmp="true"
-         data-formdata='{ "name": "Fragment {{ fragmentName }}", "type": "{{ form.vars.data.backofficeTitle }}" }'
+         data-formdata='{ "name": "{{ fragmentLabel }} {{ fragmentName }}", "type": "{{ form.vars.data.backofficeTitle|trans({}, 'SonataArticleBundle') }}" }'
```

Comment:
We are inside a piece of JSON, inside an html attribute. I think there might be escaping issues here, for instance what if there is a quote in one of the variables?